modules/repo/main.tf

```diff
@@ -16,8 +16,8 @@ variable "visibility" {
   default     = "public"
 }
 
-resource "github_repository" "this" {
-  count = var.phony ? 0 : 1
+resource "github_repository" "repo" {
+  #  count = var.phony ? 0 : 1
```

Comment:
is there a reason you commented this out, but left `phony` as an input variable?

Reply:
That's from an earlier iteration, I'll add a commit to just drop the phony thing all together.